be2-scala/src/test/scala/util/examples/JsonRpcRequestExample.scala

```diff
@@ -10,6 +10,8 @@ import util.examples.Election.OpenElectionExamples._
 import util.examples.Election.SetupElectionExamples.{ELECTION_ID, _}
 import util.examples.Election.EndElectionExamples._
 import util.examples.Election.KeyElectionExamples._
+import util.examples.Election.ResultElectionExamples.{MESSAGE_RESULT_ELECTION_TOO_MUCH_VOTES, MESSAGE_RESULT_ELECTION_WORKING, MESSAGE_RESULT_ELECTION_WRONG, MESSAGE_RESULT_ELECTION_WRONG_BALLOT_OPTIONS, MESSAGE_RESULT_ELECTION_WRONG_ID}
```

Comment:
```suggestion
import util.examples.Election.ResultElectionExamples._
```
Its quite cumbersome ..